lib/AvatarPresence/styles.scss

```diff
@@ -0,0 +1,101 @@
+/* ==========================================================================
+   Config
+   ========================================================================== */
+$avatar-size: 40px !default;
+$avatar-border-radius: 40px !default;
+$avatar-tooltip-name-weight: $typo-weight-semi-bold !default;
+$avatar-tooltip-name-size: $typo-size-lead !default;
+
+$avatar-show-more-background: rgb(175, 165, 175);
+$avatar-faded-background: rgb(60, 50, 60);
+$avatar-shadow: 0 0 0 1px rgba(100, 100, 100, 0.2) inset;
+$avatar-pillbox-padding: $typo-size-lead/3 $typo-size-lead/2 !default;
+
+/* ==========================================================================
+   Styles
+   ========================================================================== */
+.avatar {
+  border-radius: $avatar-border-radius;
+  width: $avatar-size;
+  height: $avatar-size;
+}
+
+.avatar--presence {
+  display: inline-block;
+  border-radius: 50%;
```

Comment:
Yeap, I'm replacing the original styles with these new ones anyway, so no more `avatar--presence` :)